fiftyone/utils/cvat.py

```
@@ -3358,11 +3358,13 @@ def upload_annotations(self, samples, anno_key, launch_editor=False):
 
         return results
 
-    def download_annotations(self, results):
+    def download_annotations(self, results, coerce_text_attrs=False):
```

Ruff (0.15.7)

[warning] Boolean default positional argument in function definition

(FBT002)